kuma/dashboards/tests/test_views.py

```diff
@@ -89,8 +93,58 @@ def test_spam_submission_buttons(self):
 
         page = pq(response.content)
         ip_button = page.find('td.dashboard-spam')
+        # Revisions available, admin has privileges to see this
         ok_(len(ip_button) > 0)
 
+    def test_submit_akismet_spam_post_required(self):
+        url = reverse('dashboards.submit_akismet_spam', locale='en-US')
+        response = self.client.get(url)
+        eq_(response.status_code, 405, "GET should not be allowed.")
+
+    def test_submit_akismet_spam_valid_response(self):
+        urlquery = '?' + urllib.urlencode({'page': 3})
+        urlnext = reverse('dashboards.revisions', locale='en-US') + urlquery
+        revision = Revision.objects.first()
+        data = {
+            'revision': revision.pk,
+            'submit': u'spam',
+            'next': urlnext
+        }
+        p1 = Permission.objects.get(codename='add_revisionakismetsubmission')
+        testuser = User.objects.get(username='testuser')
+        testuser.user_permissions.add(p1)
+        self.client.login(username='testuser', password='testpass')
+
+        # Response should redirect back to the revisions dash
+        urlpost = reverse('dashboards.submit_akismet_spam', locale='en-US')
+        response = self.client.post(urlpost, data=data)
+        eq_(response.status_code, 302)
+        eq_(response.url, 'http://testserver' + urlnext)
+
+        # 1 RevisionAkismetSubmission record should exist for this revision
+        ras = RevisionAkismetSubmission.objects.filter(revision=revision)
```

Comment:
@safwanrahman's right, and now I can't un-see it.  If you have time Monday morning, please fix, but I'll merge and deploy before the standup on Monday with or without it.  There's worse than this in the existing tests...